ReportingServicesTools/Functions/CatalogItems/Update-RsSubscription.ps1

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+
+function Update-RsSubscription
+{
+  <#
+        .SYNOPSIS
+            This script will update subscriptions piped from get-RsSubscriptions

```

Comment:
Seems to be the function only updates StartDateTime and EndDate , it will be great if you can actually specify the full set of settings of the subscription instead of just those two parameters.

If you decide to only do those two you might want to change the name of the function to reflect that something like
Set-RsSubscriptionDates

Reply:
adding owner as well